src/core/ext/transport/binder/java/io/grpc/binder/cpp/NativeConnectionHelper.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package io.grpc.binder.cpp;
+
+import android.content.Context;
+import android.os.IBinder;
+import android.os.Parcel;
+
+/** Provide interfaces for native code to invoke */
```

Comment:
Could you document how user should use this helper class? For example, after user call tryEstablishConnection, when should user call getServiceBinder?

Reply:
Since tryEstablishConnection could fail, should we return something to let user know?

Reply:
> Could you document how user should use this helper class? For example, after user call tryEstablishConnection, when should user call getServiceBinder?

Added some comments. Actually user will not call this class. This class is only invoked from our implementation in gRPC

> Since tryEstablishConnection could fail, should we return something to let user know?

Added a TODO. For now we just assume it always success in our prototype.